ansible_collections/arista/avd/molecule/eos_designs-mpls-isis-sr-ldp/documentation/fabric/MPLS_CORE-topology.csv

```diff
@@ -3,8 +3,10 @@ pe,SITE1-LER1,Ethernet1,p,SITE1-LSR1,Ethernet1,True
 pe,SITE1-LER1,Ethernet2,pe,SITE1-LER2,Ethernet2,True
 pe,SITE1-LER1,Ethernet3,point_to_point_service,,,True
 pe,SITE1-LER1,Ethernet4,point_to_point_service,,,True
-pe,SITE1-LER1,Ethernet6,point_to_point_service,,,True
+pe,SITE1-LER1,Ethernet6,l3_interface,,,True
```

Comment:
this changes because you added subinterfaces under a port-based service.

Reply:
This change should be reversed now.